rpc/src/module/experiment.rs

```diff
@@ -0,0 +1,82 @@
+use crate::error::RPCError;
+use ckb_core::cell::{resolve_transaction, CellProvider, CellStatus};
+use ckb_core::transaction::{OutPoint, Transaction as CoreTransaction};
+use ckb_shared::chain_state::ChainState;
+use ckb_shared::shared::Shared;
+use ckb_store::ChainStore;
+use ckb_verification::ScriptVerifier;
+use jsonrpc_core::{Error, Result};
+use jsonrpc_derive::rpc;
+use jsonrpc_types::Transaction;
+use numext_fixed_hash::H256;
+use serde_derive::Serialize;
+use std::convert::TryInto;
+
+#[rpc]
+pub trait ExperimentRpc {
+    #[rpc(name = "_compute_transaction_id")]
+    fn compute_transaction_id(&self, tx: Transaction) -> Result<H256>;
+
+    #[rpc(name = "dry_run_transaction")]
+    fn dry_run_transaction(&self, _tx: Transaction) -> Result<DryRunResult>;
+}
+
+pub(crate) struct ExperimentRpcImpl<CS> {
+    pub shared: Shared<CS>,
+}
+
+impl<CS: ChainStore + 'static> ExperimentRpc for ExperimentRpcImpl<CS> {
+    fn compute_transaction_id(&self, tx: Transaction) -> Result<H256> {
+        let tx: CoreTransaction = tx.try_into().map_err(|_| Error::parse_error())?;
+        Ok(tx.hash())
+    }
```

Comment:
`transaction_hash` better than `transaction_id` ?

Reply:
Not sure about it. Use `transaction_hash` ?

Reply:
`transaction_hash` +1